std/testing/bench_test.ts

```diff
@@ -1,7 +1,6 @@
 const { test } = Deno;
 import { bench, runBenchmarks } from "./bench.ts";
-
-import "./bench_example.ts";
```

Comment:
@nayeemrmn I removed this import, because the bench methods inside changed what I should assert.
Is it ok? Leaving it in, would mean changeing example could ruin the tests too.